homeassistant/components/nissan_leaf/sensor.py

```diff
@@ -18,9 +18,6 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Sensors setup."""
-    _LOGGER.debug("setup_platform nissan_leaf sensors, discovery_info=%s",
-                  discovery_info)
-
     devices = []
```

Comment:
Add a guard clause here that checks if discovery_info is None and return if so.

Reply:
Added guard clause in commit https://github.com/home-assistant/home-assistant/pull/21145/commits/27d28b0f10cbe70f5f25a3d0b6405dc951b5cc42